src/test/java/io/r2dbc/postgresql/PostgresqlAuthenticationFailureTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
```

Comment:
Nit: License header date reflects the inception of a file. So in this case `2019` is sufficient.